style.css

```diff
@@ -4828,6 +4828,7 @@ a.to-the-top > * {
 		transform: rotate(-180deg);
 		-ms-transform: rotate(-180deg); /* IE 9 */
```

Comment:
We don't support IE9, was that added by Autoprefixer? If not added by Autoprefixer this will get removed by Autoprefixer.

Reply:
@swapnildhanrale I just ran the code using the Autoprefixer that had been added with https://github.com/WordPress/twentytwenty/pull/758 and 

```
button.sub-menu-toggle.active svg {

	/*rtl:ignore*/
	transform: rotate(180deg);
}
```

in `style.css` will be rendered in `style-rtl.css` to

```
button.sub-menu-toggle.active svg {
	transform: rotate(180deg);
}
```

With that in mind, I'd say that the final code, to fix this issue, should be

```
button.sub-menu-toggle.active svg {

	/*rtl:ignore*/
	transform: rotate(180deg);
}
```